<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QCentralWidget{
	background-color:black;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>50</horstretch>
          <verstretch>7</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabBar::tab{
	width:100px;
}
QTabWidget::pane{
border: 0 solid white;
    margin: -13px -9px -13px -9px;
}</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab_1">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="cursorMode">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	background-color:#474747;
	border:0px solid blue;
	margin:0px 0px 0px 0px;
	padding: 0px 0px 0px 0px;
}
QPushButton:pressed
{
	border:2px solid #54bae5;
	margin:3px;
}
QPushButton:hover{
	background-color:#504f4f;
	border:2px solid #767676;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/buttons/img/cursor.png</normaloff>:/buttons/img/cursor.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bridgeMode">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>11111111</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	background-color:#474747;
	border:0px solid blue;
	margin:0px 0px 0px 0px;
	padding: 0px 0px 0px 0px;
}
QPushButton:pressed
{
	border:2px solid #54bae5;
	margin:3px;
}
QPushButton:hover{
	background-color:#504f4f;
	border:2px solid #767676;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/buttons/img/Bridge.png</normaloff>:/buttons/img/Bridge.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>70</width>
                <height>70</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="edgeMode">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	background-color:#474747;
	border:0px solid blue;
	margin:0px 0px 0px 0px;
	padding: 0px 0px 0px 0px;
}
QPushButton:pressed
{
	border:2px solid #54bae5;
	margin:3px;
}
QPushButton:hover{
	background-color:#504f4f;
	border:2px solid #767676;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/buttons/img/Edge.png</normaloff>:/buttons/img/Edge.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>70</width>
                <height>70</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="MyGraphicsSceneView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>100</horstretch>
          <verstretch>30</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsSceneView</class>
   <extends>QGraphicsView</extends>
   <header>mygraphicssceneview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
